rb_ws/src/buggy/scripts/2d_sim/engine.py

```diff
@@ -121,7 +117,7 @@ def dynamics(self, state, v):
             dstate (np.Array): time derivative of state from dynamics
         """
         l = Simulator.WHEELBASE
-        x, y, theta, delta = state
+        theta, delta = state
```

Comment:
state is a 4-member tuple, so assigning it like this will not work. Since x and y are unused, pylint was probably angry. To make pylint happy, do _, _, theta, delta instead.